tiled/structures/table.py

```diff
@@ -47,6 +47,15 @@ def from_pandas(cls, df):
         data_uri = B64_ENCODED_PREFIX + schema_b64
         return cls(arrow_schema=data_uri, npartitions=1, columns=list(df.columns))
 
+    @classmethod
+    def from_pydict(cls, d):
```

Comment:
Same as above: I suggest considering `from_dict` here.

Reply:
Changed and tested in incoming PR